src/__experimental__/components/checkable-input/hidden-checkable-input.component.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import HiddenCheckableInputStyle from './hidden-checkable-input.style';
 
 const HiddenCheckableInput = ({
-  checked, role, type, ...props
+  role, type, ...props
 }) => {
   return (
     <HiddenCheckableInputStyle
-      aria-checked={ checked }
+      aria-checked={ props.checked }
```

Comment:
What's the purpose of moving `checked` out of the destructured function parameters and using `props.checked` instead?

Reply:
Ah right, I see – by removing `checked` from the destructuring, it means `...props` now includes `checked` :thumbsup: